Classes/Command/ProjectionCommandController.php

```diff
@@ -90,12 +84,6 @@ public function describeCommand($projection): void
         $this->outputLine();
         $this->outputLine('<b>PROJECTOR:</b>');
         $this->outputLine('  %s', [$projectionDto->getProjectorClassName()]);
-        $this->outputLine();
-
-        $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
```

Comment:
We don't know the handled events at this point. We could use the new `DefaultMappingProvider` here, but I don't think that this is used a lot actually